# Devin Koepl

bool hold_robot
bool pause_simulation
bool xIsConstrained
bool yIsConstrained
bool zIsConstrained
geometry_msgs/Pose desired_pose
---
geometry_msgs/Pose actual_pose
